go/tools/builders/protoc.go

```diff
@@ -57,22 +69,41 @@ func run(args []string) error {
 		}
 	}
 	if len(notFound) > 0 {
+		missing := []string{}
```

Comment:
It doesn't seem like anything appends to this.

Reply:
Was missing a commit that I merged to the wrong branch that improved error checking
when I was merging it though I decided to re-write the whole loop to be more thorough and easier to understand (I hope)